client/landing/stepper/declarative-flow/internals/steps-repository/hundred-year-plan-step-wrapper/index.tsx

```diff
@@ -245,6 +245,27 @@ function InfoColumn( { isMobile, openModal }: { isMobile: boolean; openModal: ()
 	);
 }
 
+/**
+ * Preload videos so that they are cached in the user's browser
+ */
+const VideoPreload = memo( ( { isMobile }: { isMobile: boolean } ) =>
```

Comment:
I'm wondering if we could export this component from `client/components/hundred-year-loader-view.tsx`? This would make sure that the URLs live in one place and any updates to the video would not need to be duplicated.